tests/cross_functional/system_test/test_nfs_inout_export.py

```
@@ -0,0 +1,2081 @@
+
+        return {"pod_data": pod_data, "nfs_data": nfs_data}
+
+    @tier1
+    @pytest.mark.parametrize(
+        "access_mode",
+        [
+            # pytest.param(constants.ACCESS_MODE_RWX, id="RWX"),
+            pytest.param(constants.ACCESS_MODE_RWO, id="RWO"),
+        ],
+    )
+    def test_cluster_inout_cluster_nfs_export(
```

Ruff (0.15.15)

[warning] Too many branches (32 > 12)

(PLR0912)